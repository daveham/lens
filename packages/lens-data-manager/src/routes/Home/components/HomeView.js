import React from 'react';

import styles from './HomeView.scss';

export const HomeView = () => {
  return (
    <div className={styles.container}>
    </div>
  );
};

export default HomeView;
