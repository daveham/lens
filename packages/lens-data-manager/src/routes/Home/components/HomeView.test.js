import React from 'react';
import renderer from 'react-test-renderer';
import HomeView from './HomeView';

test('HomeView', () => {
  const component = renderer.create(
    <HomeView>test</HomeView>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
