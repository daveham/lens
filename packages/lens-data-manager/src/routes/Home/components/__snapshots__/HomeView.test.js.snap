// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeView 1`] = `
<div
  className={undefined}
/>
`;
